src/elements/content-sidebar/activity-feed/comment/Comment.scss

```diff
@@ -16,6 +16,14 @@
     }
 }
 
+.bcs-Comment-headline {
```

Comment:
Should we apply this to the UserLink component directly, so it's fixed for all usages?

Reply:
@jstoffan for some reason that didn't work. It only worked on the `<div>` above it.

Reply:
Actually it also worked if I did it on the entire `<Media.Body>` too, but that seems worse with potentially more awkward side effects for Edge.